docs/sections/user_guide/cli/drivers/mpas.rst

```diff
@@ -0,0 +1,109 @@
+``mpas``
+==========
+
+The ``uw`` mode for configuring and running the MPAS forecast model. Any of the ``TASK`` arguments listed below may be called to generate that component required for the ultimate ``run`` task. A ``provisioned_run_directory`` will perform all of the tasks except for running the executable.
+
+.. code-block:: text
+
+   $ uw mpas --help
+   usage: uw mpas [-h] [--version] TASK ...
+
+   Execute MPAS tasks
+
+   Optional arguments:
+     -h, --help
+         Show help and exit
+     --version
+         Show version info and exit
+
+   Positional arguments:
+     TASK
+       boundary_files
+         Boundary condition files
+       files_copied
+         Files copied for run
+       files_linked
+         Files linked for run
+       namelist_file
+         The namelist file
+       provisioned_run_directory
+         Run directory provisioned with all required content
+       run
+         A run
+       runscript
+         The runscript
+       streams_file
+         The streams file
+
+
+All tasks take the same arguments. For example:
+
+.. code-block:: text
+
+   $ uw mpas run --help
+   usage: uw mpas run --config-file PATH --cycle CYCLE [-h] [--version] [--batch] [--dry-run]
+                   [--graph-file PATH] [--quiet] [--verbose]
+
+   A run
+
+   Required arguments:
+     --config-file PATH, -c PATH
+         Path to config file
+     --cycle CYCLE
+         The cycle in ISO8601 format
+
+   Optional arguments:
+     -h, --help
+         Show help and exit
+     --version
+         Show version info and exit
+     --batch
+         Submit run to batch scheduler
+     --dry-run
+         Only log info, making no changes
+     --graph-file PATH
+         Path to Graphviz DOT output [experimental]
+     --quiet, -q
+         Print no logging messages
+     --verbose, -v
+         Print all logging messages
+
+
+Examples
+^^^^^^^^
+
+The examples use a configuration file named ``config.yaml`` with content similar to:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/mpas.yaml
+
+
+Its contents are described in depth in section :ref:`mpas_yaml`.
+
+* Run ``mpas`` on an interactive node
```

Comment:
We write `mpas` in several places here, but I'm not sure it's clear what that refers to. It's the name of our driver, but don't we mean the actual MPAS executable? If so, that's called `atmosphere_model` in the config, but should we maybe just write "MPAS" here and be somewhat generic?

Reply:
I think it makes sense to refer to MPAS when we want to refer to the executable or the model in general. I will try to go back in and make sure that I use driver in areas where that's what I'm referring to.